app/ilmoweb/models.py

```diff
@@ -28,6 +28,7 @@ class Labs(models.Model):
     name = models.CharField(max_length = 100)
     description = models.TextField()
     max_students = models.IntegerField()
+    minim_students = models.IntegerField(default=2)
```

Comment:
I think this is not needed. If we want to be compatible with the original database, the models should be close to the existing database. 
![image](https://github.com/ILMOWEB/ilmo/assets/75630900/2baee79f-e1ae-4239-ba40-a21d67808b78)